src/popup/container.js

```diff
@@ -66,6 +69,15 @@ class PopupContainer extends Component {
         this.getInitPauseState().then(updateState).catch(noop)
         this.getInitBlacklistBtnState(currentTab.url).then(updateState).catch(noop)
         this.getInitArchiveBtnState(currentTab.url).then(updateState).catch(noop)
+        this.getArchiveState()
+    }
+
+    async getArchiveState() {
```

Comment:
Generally only need to mark a function (or method in this case) as `async` if you use `await` to wait on Promises. Here you're just using `.then` to act on the resolution, so it's not needed. 

Also no need to reassign `this`, as you're using an ES6 fat arrow function which will inherit the scope of the caller. So this should be fine here:
```
getArchiveState() {
    browser.storage.local.get(FREEZE_DRY_ARCHIVE_KEY).then(storage =>
        this.setState(state => ({ ...state, isArchiveReady: storage[FREEZE_DRY_ARCHIVE_KEY] })))
}
```